apps/docs/src/app/platform/component-docs/platform-forms/platform-input/platform-input-example/platform-input-auto-complete-validation-example.component.ts

```diff
@@ -29,32 +30,35 @@ export class PlatformInputAutoCompleteValidationExampleComponent implements OnIn
     @Input()
     open: boolean;
 
-    /**
-     * The template with which to display the individual listed items.
-     * Use it by passing an ng-template with implicit content. See examples for more info.
-     */
-    @Input()
-    itemTemplate: TemplateRef<any>;
+    @ViewChild('typeahead')
+    typeahead: PopoverComponent;
 
     constructor(private fb: FormBuilder) {}
 
     ngOnInit(): void {
-        this.options = this.sportsData;
+        this.options = [];
     }
 
     filter(value: string): string[] {
         const filterValue = value.toLowerCase();
+        if (filterValue.length === 0) {
+            return [];
+        }
         return this.sportsData.filter((item: string) => item.toLowerCase().includes(filterValue));
     }
 
     onSearchChange(): void {
         this.options = this.filter(this.inputText);
-        this.open = true;
+        if (this.options.length > 0) {
+            this.typeahead.open();
+        }
+        this.typeahead.updatePopover();
```

Comment:
Let's do not use `this.typeahead.updatePopover();` because it reveals the problems with our popover :)

As far as I can see the problem here is connected to `popperjs` positioning in case when we do not use arrow and have empty dropdown.

To fix it we have to add css rule
![image](https://user-images.githubusercontent.com/3263744/93313595-5b699500-f811-11ea-8c0a-b2ab813b9bef.png)

But it should be done in a separate scope I believe.

Reply:
I think in scope of this PR we could just remove `this.typeahead.updatePopover()`

Reply:
> I also needed to force a popover update, because when the popover is positioned above the input, and the list is shortened via filter, the popover wasn't repositioning and it became "detached" from the input field

But this issue is still valid, looks like we have to do `this.typeahead.updatePopover()` on each change.